website/thaliedje/views.py

```diff
@@ -29,6 +29,9 @@ def get_context_data(self, **kwargs):
 
         context["venue"] = player.venue
 
+        context[
+            "has_override_permission"
```

Comment:
I would refactor this to something that better describes what it does. It’s not an `override` permission, it’s a permission to request albums and playlists.